### ISSUE TITLE
Better cpu

### ISSUE THREAD

**Issue by LiftenMH** [Minehut/Meta#668]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
both

**Describe the feature**
Better cpus from server plans

**How the feature is useful**
Allows a server to be able to handle more players with better plans, as i'm pretty sure we are stuck with the same CPU from MH10. Would be helpful to increase TPS on servers with 40+ players

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Reimagining** [comment]
Yes, many other hosts provide this. It shouldnt just be RAM we are paying for. It lowers incentive for people to take Minehut seriously.

**Comment by AaronBoiButtz** [comment]
Yea, paid servers to tend to lag the same as the MHfree server which is a total dud.
